core/migrations/0046_repository_coverage_enabled.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.7 on 2024-01-15 20:36
+
+from django.db import migrations, models
+from shared.django_apps.migration_utils import RiskyAddField, RiskyRunSQL
+
+
+class Migration(migrations.Migration):
+    """
+    BEGIN;
+    --
+    -- Add field coverage_enabled to repository
+    --
+    ALTER TABLE "repos" ADD COLUMN "coverage_enabled" boolean DEFAULT false NULL;
+    ALTER TABLE "repos" ALTER COLUMN "coverage_enabled" DROP DEFAULT;
+    --
+    -- Raw SQL operation
+    --
+    UPDATE repos SET coverage_enabled=true WHERE active=true;
+    COMMIT;
+    """
+
+    dependencies = [
+        ("core", "0045_repository_languages_last_updated"),
+    ]
+
+    operations = [
+        RiskyAddField(
+            model_name="repository",
+            name="coverage_enabled",
+            field=models.BooleanField(default=False, null=True),
+        ),
+        RiskyRunSQL("UPDATE repos SET coverage_enabled=true WHERE active=true;"),
```

Comment:
Did you confirm these run sequentially? Would bork the DB if you rand the update before the field is in there

Reply:
Yep it runs sequentially on my local migrations run, I hope the ordering is not non-deterministic, that would be crazy. Didn't find in the Django docs that mentioned ordering, but there's many instances in the code that would imply this runs sequentially.